contentcuration/contentcuration/frontend/shared/data/resources.js

```diff
@@ -541,6 +541,14 @@ class IndexedDBResource {
       collection = collection.filter(filterFn);
     }
     if (paginationActive) {
+      // Default pagination field is 'lft'
```

Comment:
You are making updates here, but there are no updates to make this configurable in the API itself. This will work if all of the results have already been loaded into IndexedDB, but if you are loading the trash page for the first time, I am not sure how this is meant to work.

See here how the pagination for contentnodes is hardcoded to be by `lft`: https://github.com/learningequality/studio/blob/unstable/contentcuration/contentcuration/viewsets/contentnode.py#L670

Reply:
done trying to fix it in backend but not sure did i did it right or not